tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -130,6 +133,12 @@ AGENT_INIT          When set to "yes", use ${AGENT_BIN} as init process in place
                     of systemd.
                     Default value: no
 
+AGENT_POLICY_FILE   Path to the agent policy rego file to be set in the rootfs.
+                    If defined, this overwrites the default setting of the
+                    permissive policy file. The path is relative to the policy
+                    rego file directory 'src/kata-opa'.
```

Comment:
Does it really need to be in src/kata-opa?  Perhaps we can make it more flexible so that devs could have the policy file anywhere in the local filesystem.

Reply:
Updated. Thanks!